posthog/temporal/workflows/squash_person_overrides.py

```diff
@@ -471,21 +420,30 @@ async def squash_events_partition(inputs: QueryInputs) -> None:
 
         if inputs.dry_run is True:
             activity.logger.info("This is a DRY RUN so nothing will be squashed.")
-            activity.logger.info("Would have run query: %s with parameters %s", query, parameters)
+            activity.logger.info("Would have run query: %s", query)
```

Comment:
that's not really helpful now (I'd revert this part) - the parameters to this query don't seem to contain any sensitive info, do they? Only for us about how many persons we have maybe, but we don't really care about that

Reply:
I would not call dry-runs useless: input logs are still available, but for the same reason I've got nothing against reverting this